src/lib/markbind/src/parser.js

```diff
@@ -48,20 +39,16 @@ class Parser {
     return _.clone(this.missingIncludeSrc);
   }
 
-  processDynamicResources(context, html) {
-    const $ = cheerio.load(html, {
-      xmlMode: false,
-      decodeEntities: false,
-    });
+  static processDynamicResources(context, html, config) {
+    const $ = cheerio.load(html);
 
-    const { rootPath } = this;
     function getAbsoluteResourcePath(elem, relativeResourcePath) {
       const firstParent = elem.closest('div[data-included-from], span[data-included-from]');
       const originalSrc = utils.ensurePosix(firstParent.attr('data-included-from') || context);
       const originalSrcFolder = path.posix.dirname(originalSrc);
       const fullResourcePath = path.posix.join(originalSrcFolder, relativeResourcePath);
-      const resolvedResourcePath = path.posix.relative(utils.ensurePosix(rootPath), fullResourcePath);
-      return path.posix.join('{{hostBaseUrl}}', resolvedResourcePath);
+      const resolvedResourcePath = path.posix.relative(utils.ensurePosix(config.rootPath), fullResourcePath);
+      return path.posix.join(config.baseUrl || '/', resolvedResourcePath);
```

Comment:
it's already rendered at this point, so we need to change it correspondingly to the actual baseUrl